CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Usage:
 * Fix #1303: Add gradle integration test and docs for AutoTLSEnricher
 * Fix #1308: Add documentation for DeploymentConfigEnricher
 * Fix #1310: Add documentation + gradle integration test for ImageChangeTriggerEnricher
+* Fix #1311: Add documentation for ProjectEnricher
```

Comment:
if this is only for openshift, maybe specify it:
commit description: `doc(openshift-plugin): Add documentation for ProjectEnricher`
and changelog `Fix #1311: Add documentation for Openshift ProjectEnricher`
How does it sound ?